workshops/templates/base_without_navbar.html

```diff
@@ -49,6 +49,8 @@
             (<a href="https://github.com/swcarpentry/amy/commit/{{workshops_git_hash}}">{{workshops_git_short_hash}}</a>
             {{workshops_git_date}}{% if workshops_git_dirty %} dirty{% endif %})
           {% endif %}
+	  <br/>
+	  This project was funded in part by a grant from the National Science Foundation (0953694) to Prof. Ethan White.
```

Comment:
I'd link to Ethan's website, but it's not working? http://whitelab.weecology.org/ cc @ethanwhite